meta/meta.go

```diff
@@ -1223,6 +1272,33 @@ func (m *Meta) UpdateDDLReorgHandle(job *model.Job, startKey, endKey kv.Key, phy
 	return errors.Trace(err)
 }
 
+// ClearALLDDLReorgHandle clears all reorganization related handles.
+func (m *Meta) ClearALLDDLReorgHandle() error {
```

Comment:
```suggestion
func (m *Meta) ClearAllDDLReorgHandle() error {
```

Golang uses camelCase.